adhocteam/Head-Start-TTADP
# yarn upgrades

**Description of change**
Yarn audit found 4 low level vulnerabilities resulting in an `EXIT_CODE` of 4 (1x4=4).  Our intention was to only fail the audit for moderate and higher vulnerabilities (4+). Unfortunately however, the `EXIT_CODE` from `yarn audit` appears to be a summation of all vulnerabilities, not a statement of the max vulnerability. This is an irritating problem not addressed by this PR. This sidesteps the bigger problem and just upgrades packages to get rid of the current low level vulnerabilities.

**How to test**
See before (fail) and after (pass) of audit job on circle ci https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP?branch=sj-yarn-upgrade

**Issue(s)**

**Checklist**

- [n/a] Meets issue criteria
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
